phpmyfaq/src/phpMyFAQ/Controller/Administration/FaqController.php

```diff
@@ -277,4 +271,62 @@ public function saveOrderOfStickyFaqs(Request $request): JsonResponse
 
         return $response;
     }
+
+    #[Route('admin/api/faq/import')]
+    public function importFaqs(Request $request): JsonResponse {
+        $response = new JsonResponse();
+        
+        $file = $_FILES['file'];
+        if(!isset($file)) {
+            $response->setStatusCode(Response::HTTP_BAD_REQUEST);
```

Comment:
We should add an error message here as well.

Reply:
I used only an english message. Normally, it is not able, that no file is submitted.